atoma-proxy/src/server/mod.rs

```diff
@@ -156,7 +156,7 @@ pub async fn chat_completions_handler(
     let max_tokens = payload
         .get("max_tokens")
         .and_then(|max_tokens| max_tokens.as_u64())
-        .ok_or(StatusCode::BAD_REQUEST)?;
+        .unwrap_or(STACK_ENTRY_COMPUTE_UNITS);
```

Comment:
where is this variable comes from ?

Reply:
from the const, that's the variable we currently create stacks with